pkg/kubehound/graph/edge/role_bind_crb_cr_r.go

```diff
@@ -167,12 +170,47 @@ func (e *RoleBindCrbCrR) Stream(ctx context.Context, store storedb.Provider, c c
 			},
 		},
 		// Looking for all permissionSets link to the same namespace
+		// {
+		// 	"$lookup": bson.M{
+		// 		"as":   "rb",
+		// 		"from": "rolebindings",
+		// 		"let": bson.M{
+		// 			"roleId": "$_id",
+		// 		},
+		// 		"pipeline": []bson.M{
+		// 			{
+		// 				"$match": bson.M{
+		// 					"$or": bson.A{
+		// 						bson.M{"$expr": bson.M{
+		// 							"$eq": bson.A{
+		// 								"$k8.objectmeta.namespace", "$$roleId",
+		// 							},
+		// 						}},
+		// 						bson.M{"is_namespaced": false},
+		// 					},
+		// 					"runtime.runID":   runtime.RunID.String(),
+		// 					"runtime.cluster": runtime.ClusterName,
+		// 				},
+		// 			},
+		// 			{
+		// 				"$project": bson.M{
+		// 					"_id": 1,
+		// 				},
+		// 			},
+		// 		},
+		// 	},
+		// },
+		// Looking for all permissionSets link to the same namespace
 		{
 			"$lookup": bson.M{
 				"from":         "rolebindings",
 				"localField":   "role_binding_id",
 				"foreignField": "_id",
 				"as":           "rb",
+				// "let": bson.M{
```

Comment:
Could we comment on why we don't want the cluster + run id filtering here? It's not clear to me why we don't want that here

Reply:
As we are using the _id as foreignField, we dont need to filter them as it is already filtered. So I "removed" it to make it more effecient.
Removing both commented parts.